src/cljdoc/server/search/artifact_indexer.clj

```diff
@@ -36,10 +37,17 @@
                :description (str "Clojure Contrib library " g "/" a)
                :origin :maven-central}))))
 
+(def ^:private maven-artefacts ["org.clojure"
```

Comment:
"artefacts" is misleading, correct would be maven-groups

Reply:
fixed in 
81ce58c